nltk/tokenize/stanford_segmenter.py

```diff
@@ -64,13 +65,20 @@ def __init__(self,
                 env_vars=('STANFORD_SEGMENTER',),
                 searchpath=(), url=_stanford_url,
                 verbose=verbose)
-        slf4j = find_jar(
+        if path_to_slf4j is not None:
+            if path_to_slf4j == 'PLACEHOLDER':
+                path_to_slf4j = None
+            slf4j = find_jar(
                 self._SLF4J, path_to_slf4j,
                 env_vars=('SLF4J', 'STANFORD_SEGMENTER',),
                 searchpath=(), url=_stanford_url,
                 verbose=verbose)
+        else:
+            slf4j = None
 
-        # This is passed to java as the -cp option, the segmenter needs slf4j.
+        # This is passed to java as the -cp option, the old version of segmenter needs slf4j.
+        # The new version of stanford-segmenter-2016-10-31 doesn't need slf4j
```

Comment:
Yes, it looks so. Could you please document this behavior and mention that if an old version of the segmenter is used, then the `path_to_slf4j` has to be set and a good value for it is `'slf4j-api.jar'`.

Also line 49 `_SLF4J = 'slf4j-api.jar'` needs to be cleaned up.